pkg/vault/vault_client.go

```diff
@@ -145,10 +145,17 @@ func (v *client) List(path string) ([]string, error) {
 	if secrets == nil {
 		return secretNames, nil
 	}
+	
 	data := secrets.Data
 	if data == nil {
 		return secretNames, nil
 	}
+  
+  // Don't do type assertion on nil
```

Comment:
I think the formatting is off here. You can run `make fmt` to check and fix formatting